src/inspector/models/notation/lines/textlinesettingsmodel.cpp

```diff
@@ -74,9 +74,15 @@ void TextLineSettingsModel::createProperties()
     m_dashLineLength = buildPropertyItem(Pid::DASH_LINE_LEN);
     m_dashGapLength = buildPropertyItem(Pid::DASH_GAP_LEN);
 
-    m_hookHeight = buildPropertyItem(Pid::END_HOOK_HEIGHT, [this](const Pid pid, const QVariant& newValue) {
+    // m_hookHeight = buildPropertyItem(Pid::END_HOOK_HEIGHT, [this](const Pid pid, const QVariant& newValue) {
+    //     onPropertyValueChanged(pid, newValue);
+    //     onPropertyValueChanged(Pid::BEGIN_HOOK_HEIGHT, newValue);
+    // });
+    m_startHookHeight = buildPropertyItem(Pid::BEGIN_HOOK_HEIGHT, [this](const Pid pid, const QVariant& newValue) {
+        onPropertyValueChanged(pid, newValue);
+    });
+    m_endHookHeight = buildPropertyItem(Pid::END_HOOK_HEIGHT, [this](const Pid pid, const QVariant& newValue) {
         onPropertyValueChanged(pid, newValue);
```

Comment:
Since these PropertyItems are now normal (i.e. just one property per property item, instead of two properties in one item), it is not necessary anymore to specify the "setter" explicitly. So you can simplify this code to:
```c++
m_startHookHeight = buildPropertyItem(Pid::BEGIN_HOOK_HEIGHT);
m_endHookHeight = buildPropertyItem(Pid::END_HOOK_HEIGHT);